commitlint/grammar.md

```diff
@@ -0,0 +1,112 @@
+# Mender conventional commit specification
+
+```
+<type>(<scope>): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+# Specification
+
+    1. Commits MUST be prefixed with a type, which consists of a noun, feat, fix, etc., followed by the OPTIONAL scope, and REQUIRED terminal colon and space (See #allowed-types for more information).
+    2. The type feat MUST be used when a commit adds a new feature to your application or library.
+    3. The type fix MUST be used when a commit represents a bug fix for your application.
+    4. A scope MAY be provided after a type. A scope MUST consist of a noun describing a section of the codebase surrounded by parenthesis, e.g., fix(parser):
+    5. A description MUST immediately follow the colon and space after the type/scope prefix. The description is a short summary of the code changes, e.g., fix: array parsing issue when multiple spaces were contained in string.
+    6. A longer commit body MAY be provided after the short description, providing additional contextual information about the code changes. The body MUST begin one blank line after the description.
+    7. A commit body is free-form and MAY consist of any number of newline separated paragraphs.
+    8. One or more footers MAY be provided one blank line after the body. Each footer MUST consist of a word token, followed by a :<space> separator, followed by a string value (this is inspired by the git trailer convention). See #allowed-footers for more information.
+    9. A footer’s token MUST use - in place of whitespace characters, e.g., Acked-by (this helps differentiate the footer section from a multi-paragraph body). An exception is made for BREAKING CHANGE, which MAY also be used as a token.
```

Comment:
Does this mean that if I misspell and type "BREKING CHANGE", or even lowercase "Breaking change", this will trigger an error? That's probably a good thing if it does, I'm just making sure.

Reply:
Yes :)